11-licensing.md

```diff
@@ -47,6 +47,7 @@ the most popular, including the following:
   (GPL),
 * [MIT license](http://opensource.org/licenses/MIT),
 * [BSD license](http://opensource.org/licenses/BSD-2-Clause).
+* [Apache license, Version 2.0](http://www.apache.org/licenses/LICENSE-2.0)
```

Comment:
How about linking to the opensource.org hosted version: https://opensource.org/licenses/Apache-2.0 ?

Before finding it on opensource.org, I wasn't sure if Apache license was an Open Source license or not.